hare.spec

```diff
@@ -91,11 +95,13 @@ groupadd --force hare
 chgrp hare /var/lib/hare
 chmod --changes g+w /var/lib/hare
 
+%if %{rhel} < 8
 # puppet-agent provides a newer version of facter, but sometimes it might not be
```

Comment:
Can this mentioned thing in the comment happen with CentOs ver =>8 (sometimes it might not be available at path ...)?
So I was wondering, can it be more risk free if we do not include this added `%if %{rhel} < 8` condition here.
Thanks!

Reply:
No, it cannot happen in CentOS 8.